README.md

```diff
@@ -2,32 +2,36 @@
 ---
 ### Introduction
 A central task in drug discovery is searching, screening, and organizing large chemical databases.
-Here, we implement clustering on molecular similarity.
-We compute Morgan circular fingerprints, cluster using k-means, and perform dimensionality reduction using UMAP.
-Distributed GPU-accelerated algorithms enable real-time interactive exploration of chemical space.
+Here, we implement clustering on molecular similarity. We support multiple methods to provide a interactive exploration of chemical space.
 
-![screenshot](screenshot.jpg "Screenshot of cheminformatics dashboard")
+ - Compute Morgan circular fingerprints, cluster using k-means, and perform dimensionality reduction using UMAP. Distributed GPU-accelerated algorithms with multi-GPU support are used in this method. This allows processing very large datasets.
+ - Compute Morgan circular fingerprints, [Sparse Random Projection](https://docs.rapids.ai/api/cuml/stable/api.html?highlight=sparserandomprojection#cuml.random_projection.SparseRandomProjection) and cluster using k-means.
+
+![screenshot](screenshot.png "Screenshot of cheminformatics dashboard")
 
 ### Preparing your environment (optional)
-A launch script, `launch.sh`, is provided to perform all tasks. 
+A launch script, `launch.sh`, is provided to perform all tasks.
 
 #### Customization
 
-The environment can be customized for control of the container, to create your own repo, or to store the data in a custom location. However, if this is not needed, 
+The environment can be customized for control of the container, to create your own repo, or to store the data in a custom location. However, if this is not needed,
 skip to [Getting Started](#getting-started) to use the defaults.
 
-To customize your local environment, edit the appropriate section of `launch.sh` or provide a `~/.cheminf_local_environment` file with the following information below.  
-To generate a template for `.cheminf_local_environment`, just run `./launch.sh` with no arguments.  
+To customize your local environment, edit the appropriate section of `launch.sh` or provide a `~/.cheminf_local_environment` file with the following information below.
+To generate a template for `.cheminf_local_environment`, just run `./launch.sh` with no arguments.
 If `.cheminf_local_environment` does not exist, then a template will be written for you.
 
 ```
-CONT=gitlab-master.nvidia.com:5005/<path_to_your_repo>
+CONT=gitlab-master.nvidia.com:5005/cheminf-clustering
 REGISTRY=gitlab-master.nvidia.com:5005
 REGISTRY_USER='$oauthtoken'
 REGISTRY_ACCESS_TOKEN=$(cat <path_to_your_access_token>)
```

Comment:
This is an NGC registry token, so we will probably need to mention that and provide a link to the setup stuff for NGC. I can help with this if needed.